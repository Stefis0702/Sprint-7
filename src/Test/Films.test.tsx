import FilmList from "../components/Films/Films";

import{describe,it, expect} from 'vitest';

describe('FilmList',()=>{
    it('should be a function',()=>{
        expect( FilmList).toBeTypeOf('function');
    });
    it('should render correctly',()=>{
        expect(FilmList).toMatchSnapshot();
    });
    
    
})