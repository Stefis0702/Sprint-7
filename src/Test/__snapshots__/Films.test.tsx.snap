// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FilmList > should render correctly 1`] = `[Function]`;

exports[`PilotsList > should render correctly 1`] = `[Function]`;
